build/deps/v8.bzl

```diff
@@ -30,6 +30,7 @@ PATCHES = [
     "0022-Reset-code_range_-before-pointer-compression-cage.patch",
     "0023-Move-tear-down-in-IsolateGroup-Release-into-destruct.patch",
     "0024-Modify-where-to-look-for-fast_float-and-simdutf.patch",
+    "0025-add-processed_characters-option-to-WriteUtf8V2.patch"
```

Comment:
My guess is that the lint issue is a missing trailing comma here.
Interesting that buildifier didn't print the issue to stderr, probably some missing flag in the command line in format.py?
@npaun